CMakeLists.txt

```diff
@@ -288,9 +288,12 @@ foreach(kind ${kinds})
 
   target_include_directories(${libTgt}_f PRIVATE include
                                                  fms
+                                                 fms/include
                                                  fms2_io/include
+                                                 string_utils/include
                                                  mpp/include
-                                                 constants)
+                                                 constants
+                                                 test_fms/fms/include)
```

Comment:
not sure if test_fms is  needed for CMake yet. @rem1776, is this correct?

Reply:
Yeah that's correct we don't have cmake set up to run the tests so we don't need to compile them